### PR TITLE
Reviews needed please.

### PR DESCRIPTION
**Pull request by SanMasood**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+ */
+
+const request = require('supertest');
+const nock = require('nock');
+const app = require('../src/app');
+const mockResponse = require('../src/mockResponse.js');
+
+const api = 'https://api.icndb.com';
+const mockError = { statusCode: 500, message: 'Error responding' };
+const errFunc = res => {
```
Comment (anywhereiromy) [inline __tests__/app.test.js]: What is this function doing? the name of it doesn't make me clear what your intended use is, so think about if you could rename it to make that more clear?

Comment (SanMasood) [reply __tests__/app.test.js]: This is the 'error' function. It is invoked wherever we need the error status and message to be sent. I wasn't sure of a proper name that I should call it so I just named it errFunc for errorFunction. What should I name it?

Comment (SanMasood) [reply __tests__/app.test.js]:
```
const errFunc = res => {
  expect(res.statusCode).toEqual(500);
  expect(res.body.error).toEqual(mockError.message);
};
```

I put these statements in a function body to make them less repetitive.

Comment (ersel) [reply __tests__/app.test.js]: interesting refactor, perhaps you could name it `checkForError` `assertError` or something like that to describe the intent

**Review by breakfastmeansbreakfast — APPROVED** [review]
Much cleaner Sana. Overall looks good, if you're finished I would remove any remaining comments and add instructions around the various routes outlined in readme.MD into your public/index.html file, so people know where to go if they visit your api at its top level.

**Pull request closed without merging**